src/main/scala/com/databricks/labs/overwatch/pipeline/BronzeTransforms.scala

```diff
@@ -864,13 +867,52 @@ trait BronzeTransforms extends SparkSessionWrapper {
     }
   }
 
+
+  private[overwatch] def getAllEventLogPrefix(inputDataframe: DataFrame, apiEnv: ApiEnv): DataFrame = {
+    val mountMap = getMountPointMapping(apiEnv) //Getting the mount info from api and cleaning the data
+      .withColumn("source", when('source.endsWith("/"), 'source.substr(lit(0), length('source) - 1)).otherwise('source))
+      .filter(col("mount_point") =!= "/")
+
+    //Cleaning the data for cluster log path
+    val formattedInputDf = inputDataframe.withColumn("cluster_log_conf", when('cluster_log_conf.endsWith("/"), 'cluster_log_conf.substr(lit(0), length('cluster_log_conf) - 1)).otherwise('cluster_log_conf))
+      .withColumn("cluster_mount_point_temp", regexp_replace('cluster_log_conf, "dbfs:", ""))
+      .withColumn("cluster_mount_point", regexp_replace('cluster_mount_point_temp, "//", "/"))
```

Comment:
This breaks s3:// direct path log locations as s3:/ is not a valid uri s3:// is. Also I'm not sure what the purpose of this line is at all because s3://some_path/to///here/with/////multiple_slashes also works just fine.

cc @Sriram-databricks -- urgent required fix